plugins/rule_engines/irods_rule_language/src/configuration.cpp

```diff
@@ -420,20 +422,60 @@ int hash_rules(const std::vector<std::string> &irbs, const int pid, std::string
 
 class make_copy {
 public:
-        make_copy(const std::vector<std::string>& _irbs, const int _pid) : irbs_(_irbs), pid_(_pid) {
-            for(auto const &irb : _irbs) {
-                boost::filesystem::copy_file(get_rule_base_path(irb), get_rule_base_path_copy(irb, _pid));
-            }
+  make_copy(const std::vector<std::string>& _irbs)
```

Comment:
What does `irbs` stand for? in-memory rulebases?

Reply:
probably iRODS rule bases

Reply:
definitely do not need to keep that nomenclature... strong proposals welcome.

Reply:
I believe it's a vector of strings that correspond to the various `.re` files in `/etc/irods` (e.g. `core` for `core.re`), so if I had to hazard a guess, it'd be something like `irods rule bases`? I mostly copied the naming scheme from the existing implementation.

Reply:
Should be easy enough to check. Could use a debugger for that.

Reply:
Did a quick debug print. Seems to map 1-to-1 to the "re_rulebase_set" array in `server_config.json`. So `iRODS rule bases` definitely seems right here.

Reply:
excellent.   yes, please use all the letters for a new name, so any future readers of the code don't have to think about this again.

Reply:
Agreed.